ui/app/components/app/signature-request-original.js

```diff
@@ -273,8 +273,7 @@ SignatureRequest.prototype.renderBody = function () {
       }),
     }, [notice]),
 
-    h('div.request-signature__rows', type === 'eth_signTypedData' && version === 'V3' ?
-      this.renderTypedDataV3(data) :
```

Comment:
Should delete the `this.renderTypedDataV3` method too? No longer needed here, correct?